### ISSUE TITLE
New functionality: Images that don't change color

### ISSUE THREAD

**Issue by dormiraucourant** [JoseSoteloCohen/Blackout-WordPress-Dark-mode-Widget#29]
Your plugin is perfect but if you can prevent the images from changing colors when you put the site in "Dark Mode", it would be perfect.

For now, this is the only reason preventing me from using your plugin.

Thank you for your understanding.

NB : There are 2 images to  show you the problem :

![image1dm](https://user-images.githubusercontent.com/79178703/108143202-92e1b380-70c7-11eb-9b62-9a1aaaf0080f.PNG)
![image2dm](https://user-images.githubusercontent.com/79178703/108143210-970dd100-70c7-11eb-8e1a-557d1bc83168.PNG)

**Comment by JoseSoteloCohen** [comment]
Hello @dormiraucourant 

Thank you for reaching out, in order for me to help I need to see the site with the plugin enabled, maybe you  can do it on a staging subdomain?

I don't need access, just to see the front-end.

Cheers

**Comment by dormiraucourant** [comment]
Hi JoseSoteloCohen,

Thank you for your fast answer, I've activated the plugin, you can see it on this website : https://dormiraucourant.fr/

**Comment by dormiraucourant** [comment]
I have to desactivate the public for now, if you need I activate again, just tell it to me  below this comment ;)

**Comment by JoseSoteloCohen** [comment]
Hello @dormiraucourant 

Thank you for activating it.

This CSS code should fix it:

`.entry-image{
    mix-blend-mode: difference;
}`

You can add that in the customizer custom CSS section, or on your child theme style.css file.

Let me know if it works.

Thanks!

**Comment by dormiraucourant** [comment]
No, because now, the images change color when it's not in Dark Mode..

**Comment by JoseSoteloCohen** [comment]
hi @dormiraucourant 

Sorry for that, is hard to test without the plugin being installed, but this should do the trick:

`.darkmode--activated .entry-image{ mix-blend-mode: difference; }`

Basically, darkmode--activated makes sure that it only happens when darkmode is active.

Can you try that one?

**Comment by JustagirltryingtogetanA** [comment]
Hi @JoseSoteloCohen I'm having that same problem with my website. I tried putting in that code but nothing changed - but I'm not that great with WP so maybe it's me. The theme I'm using is Aquene, if that indicates anything. Other than that, I think it's quite good and very easy on the eyes.

**Comment by JoseSoteloCohen** [comment]
hi @JustagirltryingtogetanA can you share your website?

This plugin needs custom solution to work as every theme/site is different.

**Comment by JustagirltryingtogetanA** [comment]
Hi @JoseSoteloCohen thank you so much for the speedy reply. I shall email it to you as I would prefer to not have the site name out in the open :)

**Comment by jadenfreude** [comment]
Hi @JustagirltryingtogetanA I'm not sure if you still need this but this worked for me:
`.darkmode--activated .darkmode-ignore{
	mix-blend-mode: normal !important;
	filter: invert(1);
	-webkit-filter: invert(1);
}`

It's pretty hacky but it should work for any website (in theory).
